scripts/host_cap.py

```diff
@@ -483,7 +483,7 @@ def write_host_cap(host_model, api, module_name, output_dir, run_env):
             # Look for any loop-variable mismatch
             for suite in api.suites:
                 spart_list = suite_part_list(suite, stage)
-                for spart in spart_list:
+                for _, spart in sorted(enumerate(spart_list)):
```

Comment:
good question! I had added "sorted" to line 533 on a different branch to keep the use statements in a consistent order, which broke when you use multiple groups (thereby invalidating the whole point of the change of course), so that line became "sorted(enumerate(" and then I just migrated the full change to the other loop for... reasons that made sense at the time.

I'll revert these changes here and will bring in the sorted changes (there are are more than just this one) in a future PR.

Reply:
reverted